Lib/_collections_abc.py

```diff
@@ -1013,20 +1013,10 @@ def index(self, value, start=0, stop=None):
            Supporting start and stop arguments is optional, but
            recommended.
         '''
-        if start is not None and start < 0:
-            start = max(len(self) + start, 0)
-        if stop is not None and stop < 0:
-            stop += len(self)
-
-        i = start
-        while stop is None or i < stop:
-            try:
-                v = self[i]
-                if v is value or v == value:
-                    return i
-            except IndexError:
-                break
-            i += 1
+        for i in range(len(sequence))[start:stop]:
```

Comment:
All tests complain that `NameError: name 'sequence' is not defined`. Since we're already in `Sequence` class, maybe `sequence` should be changed to `self` (instead of hypothetical `sequence = self`)?

```suggestion
        for i in range(len(self))[start:stop]:
```

Reply:
Typo, thanks.